capstone/src/main/java/com/google/sps/servlets/ParseTestGroupDataServlet.java

```diff
@@ -80,14 +80,16 @@ private void createGroupEntity(String[] fields) {
    * Create a Post object and add it to the datastore given information from TSV file.
    */
   private long createPost(String postText, String author) {
+    String authorId = author;
     String authorName = author;
```

Comment:
Why is the authorId the same as authorName?

Reply:
Initially when I wrote the ParseTestGroupDataServlet I set authorName to be equal to the userId - I thought for the test data it was okay if the post displayed "JANE_DOE" instead of "JANE DOE". In the TSV file for group data we only have userIds to identify the members of a group, but no other user information. If we wanted to make the test data more accurate we could split up the name by "_" and set the author name that way. In the real post servlet we'll have access to all the user information so they would be different.

Reply:
I set the authorId and authorName to the same value since our test group data only has the authorId's. This is a just a workaround for the test data, but the correct authorId and authorName will be stored for a real user.

Reply:
Ah I didn't notice this was for the test data, thank y'all for the explanation!